src/ia/js/IAPage.js

```diff
@@ -2,6 +2,31 @@
     // Handlebars helpers
     Handlebars.registerHelper('encodeURIComponent', encodeURIComponent);
 
+    // Choose a GitHub location before anything else.
+    Handlebars.registerHelper('chooseService', function(services) {
+        // Make sure to build the URL if we just find the handle.
```

Comment:
@russellholt I don't think `services` can be null because `{{each}}` is already going through the array. So if `attribution` is null, `{{chooseService}}` won't be executed.